.circleci/config.yml

```diff
@@ -185,11 +185,24 @@ jobs:
             errors=$(jq '.["results"] | .[] | unique_by(.["type"]) | map(select(.["type"] == "error")) | .[]' planet4-docker-compose/artifacts/pa11y/pa11y-ci-results.json)
             if [ ! -z "${errors}" ]; then echo "Errors found, see report in artifacts." && exit 1; else echo "No errors, report available in artifacts."; fi
 
-  rips-test:
+  create-release-zip:
     docker:
-      - image: rips/rips-cli:3
-    environment:
-      RIPS_APPLICATION_ID: 21
+      - image: mhart/alpine-node
     steps:
       - checkout
-      - run: rips-cli -vvv rips:scan:start -a $RIPS_APPLICATION_ID -p ~/project/ -t 1
+      - run: apk add curl zip git jq
+      - run: git submodule init && git submodule update
+      - run: PUPPETEER_SKIP_DOWNLOAD=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true npm ci
```

Comment:
:tada:

Reply:
waaait you can do that ? This thing kills me every time I reinstall dev env
![mind blown](https://media.giphy.com/media/d0MYTsbRnGKoU/source.gif)

oooh there's even a package without chromium download https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#puppeteer-vs-puppeteer-core

Reply:
Looks like Gutenberg in fact will use that instead, I think it's also introduced in the 5.5 release. https://github.com/WordPress/gutenberg/pull/20215